File: dotnet/Lexy/Lexy.Compiler/Compiler/CSharp/ValidateTableKeyword.cs
using System.Collections.Generic;

namespace Lexy.Compiler.Compiler.CSharp
{
    public class ValidateTableKeyword
    {
        public class ValidateTableKeywordRow
        {
            public int Value { get; set; }
            public int Result { get; set; }
        }
        private static IList<ValidateTableKeywordRow> _value = new List<ValidateTableKeywordRow>();
        static ValidateTableKeyword()
        {
            _value.Add(new ValidateTableKeywordRow
            {
                Value = 0,
                Result = 0,
            });
            _value.Add(new ValidateTableKeywordRow
            {
                Value = 1,
                Result = 1,
            });
        }

        public ValidateTableKeyword()
        {
        }

        public int Count => _value.Count;
        public IEnumerable<ValidateTableKeywordRow> Value => _value;
    }
}

File: typescipt/src/compiler/cSharp/validateTableKeyword.ts


export class ValidateTableKeyword {
   private static readonly Array<ValidateTableKeywordRow> _value = list<ValidateTableKeywordRow>(): new;

   public number Count => _value.Count;
   public Array<ValidateTableKeywordRow> Value => _value;

   static ValidateTableKeyword() {
     _value.Add(new ValidateTableKeywordRow {
       Value = 0,
       Result = 0
     });
     _value.Add(new ValidateTableKeywordRow {
       Value = 1,
       Result = 1
     });
   }

   public class ValidateTableKeywordRow {
     public number Value { get; set; }
     public number Result { get; set; }
   }
}
